server/chat/backend/agent/agent.py

```
@@ -82,13 +81,12 @@ def _convert_chunk_to_generation_chunk(self, chunk, default_chunk_class, base_ge
         return result
 
 class Agent:
-    def __init__(self, weaviate_client: WeaviateClient, postgres_client: PostgreSQLClient, websocket_sender=None, event_loop=None, ctx_len=10):
+    def __init__(self, postgres_client: PostgreSQLClient, websocket_sender=None, event_loop=None, ctx_len=10):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)